webapp/roller_algo.js

```diff
@@ -380,13 +409,34 @@ function make_support_list(cardset)
 			support_cards["Ruins"] = true;
 		}
 
+		// Fate type cards use Boons, but Druid won't require the whole pile
+// 		if (c.type && c.type.match(/Fate/)) {
+		if (c.type && c.type.match(/Fate/) && c.id && !c.id == 'Druid') {
+			support_cards["Boons"] = true;
+			// One of the boons give Will-o'-Wisp
+			support_cards["Will-o'-Wisp"] = true;
```

Comment:
hmmm... adding in the 'Will-o-wisp' requirement here seems wrong.
Since any card that requires Boons will transitively require Will-o-Wisp.
I wonder if we can add requires: ["Will-o'-Wisp"] for Boons in the all-cards list, and then support transitive card requirements here.